### DISCUSSION TITLE
Option to turn on and off snap to bed

### DISCUSSION THREAD

**Discussion by Brakisshuro** [OrcaSlicer/OrcaSlicer#6481] [Ideas]
When I'm printing helmets I would prefer if the entire helmet was floating and I built support around it. So a check option to turn off snapping to bed or an option in support to be able to lift items off the bed when generating support. Would make cleanup of some items much easier.

**Comment by zaelu** [comment]
Bump this plea also! Please. It's really necessary!

**Comment by yw4z** [comment]
you can assemble it with a small box then move it to upwards
<img width="1072" height="621" alt="Screenshot-20251119222649" src="https://github.com/user-attachments/assets/2b278492-2991-4cef-8cb9-7dab585cefd7" />
